app/models/request.rb

```diff
@@ -80,8 +85,9 @@ def find_existing_user
   end
 
   def holdings
-    if persisted?
-      @holdings ||= searchworks_item.requested_holdings.by_barcodes(barcodes)
+    if persisted? || requested_barcode.present?
```

Comment:
This is just temporary, I would like to find a better way of doing this in general.